devicemodel/include/vmmapi.h

```diff
@@ -61,6 +63,10 @@ struct vmctx {
 	void *atkbdc_base;
 	void *vrtc;
 	void *ioc_dev;
+	/* cmos buffer used to store write/read contents,
+	 * and it should not be cleared when reboot
```

Comment:
"should not" or "should"?

Reply:
The data should not be cleared when reboot.
For the content for vrpmb, the data is always cleared after read.